compiler-rt/lib/fuzzer/FuzzerMutate.cpp

```diff
@@ -575,8 +584,7 @@ size_t MutationDispatcher::MutateWithMask(uint8_t *Data, size_t Size,
 }
 
 void MutationDispatcher::AddWordToManualDictionary(const Word &W) {
-  ManualDictionary.push_back(
-      {W, std::numeric_limits<size_t>::max()});
+  ManualDictionary.push_back({W, PositionHint{1, W.size()}});
```

Comment:
If I understand correctly, the old code uses `(size_t)-1` to indicate no hint, but your new code creates a valid hint. What's the reason for that change, and why is the hint index 1?

Reply:
Fixed, this was just a copy & paste error from [here](https://github.com/CodeIntelligenceTesting/llvm-project/pull/1/files#diff-3be2c27c6f384d5c2f603c6d8742878cc46b50e5aee69749dbc52ffbd2aeae93R478). I preserved the 1 there, but I have no idea why it's there. I asked on llvm-dev, but got no response. I am 99% sure this should be `std::numeric_limits<size_t>::max()` instead. What do you think?

Reply:
I agree, there doesn't seem to be a reason for constant 1, except maybe if they're trying to give small shifts a higher priority (because the index to insert or overwrite would be chosen arbitrarily if there was no hint).